scenes/game/local_game.gd

```diff
@@ -814,6 +835,20 @@ class Player:
 		gauge = []
 		return events
 
+	func swap_deck_and_sealed():
+		var events = []
+		var current_sealed_ids = sealed.map(func(card) : return card.id)
+		var current_deck_ids = deck.map(func(card) : return card.id)
+		for card_id in current_deck_ids:
+			var seal_effect = { "effect_type": "seal_card_INTERNAL", "seal_card_id": card_id, "source": "deck", "silent": true }
+			events += parent.handle_strike_effect(-1, seal_effect, self)
+		for card_id in current_sealed_ids:
+			events += move_card_from_sealed_to_top_deck(card_id)
+		parent._append_log_full(Enums.LogType.LogType_CardInfo, self, "swaps their sealed cards and deck!")
+		events += [parent.create_event(Enums.EventType.EventType_Popup, my_id, 0, "", "Swap Sealed and Deck")]
```

Comment:
Not a fan of localgame doing generic popups.
This should really be an event specific to this.